core/src/main/resources/jenkins/install/pluginSetupWizard.properties

```diff
@@ -56,3 +58,17 @@ installWizard_pluginInstallFailure_message=Some plugins failed to install proper
 with the failed plugins
 installWizard_continue=Continue
 installWizard_retry=Retry
+installWizard_upgradePanel_title=Upgrade
+installWizard_upgradePanel_banner=Welcome to Jenkins 2!
+installWizard_upgradePanel_message=Jenkins 2 includes some great new features we think you'll love to use. Get your instance updated to the latest \
+and greatest plugins available and take advantage of them!
+installWizard_upgradePanel_recommendedActionTitle=Install Recommended Plugins
```

Comment:
s/Recommended/Suggested/

The terminology should be consistent (suggested) between the upgrade and new user flow.